lib/daru/index/index.rb

```diff
@@ -222,6 +223,10 @@ def reorder(new_order)
       self.class.new(new_order.map { |i| from[i] })
     end
 
+    def set_name _name
```

Comment:
Thanks @zverok. I will keep these points in my mind. I have edited the PR accordingly. Now I am using `attr_accessor` for `name`.